zest/releaser/vcs.py

```diff
@@ -4,6 +4,7 @@
 
 import logging
 import os
+import pkg_resources
```

Comment:
This is something that will bite us in the future: pkg_resources is going away, I believe :-)

Reply:
True, but I copied this from `baserelease.py`.  And we are using `pkg_resources` in other places as well, for example for the hooks.

Reply:
I'm also using it in lots of places myself, so I was a bit scared when I saw that it was probably going to be deprecated somewhere in the future :-)

Fine to leave it in here, for now, of course.